File: 99_bottles.py
def bottle_song():
    for i in range(99, 0, -1):
        # Use f-strings for string formatting
        bottles_text = " bottle" if i == 1 else " bottles"

        print(f"{i}{bottles_text} of beer on the wall, {i}{bottles_text} of beer.")

        if i > 1:
            print(
                f"Take one down and pass it around, {i - 1}{' bottle' if i - 1 == 1 else ' bottles'} of beer on the wall."
            )
        else:
            print(
                "Take one down and pass it around, no more bottles of beer on the wall."
            )


# Call the function to execute the song
bottle_song()

File: 99_bottles.js
function bottleSong() {
  for (var i = 99; i > 0; i--) {
    // Use ternary operator to handle singular and plural forms
    var bottlesText = i === 1 ? " bottle" : " bottles";

    console.log(
      i + bottlesText + " of beer on the wall, " + i + bottlesText + " of beer."
    );

    if (i > 1) {
      console.log(
        `Take one down and pass it around, ${i - 1}${
          i - 1 === 1 ? " bottle" : " bottles"
        } of beer on the wall.`
      );
    } else {
      console.log(
        "Take one down and pass it around, no more bottles of beer on the wall."
      );
    }
  }
}

bottleSong();
